00:39:54 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:40:59 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:42:52 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:40:31 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:40:56 - main - ERROR - <module>: 219 - A request to the Telegram API was unsuccessful. Error code: 400. Description: Bad Request: too many messages to send as an album
Traceback (most recent call last):
  File "/home/runner/work/FortDaily-PVE_bot/FortDaily-PVE_bot/main.py", line 216, in <module>
    sending()
  File "/home/runner/work/FortDaily-PVE_bot/FortDaily-PVE_bot/main.py", line 185, in sending
    bot.send_media_group(chat_id=channel_id, media=mediagroup)
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/telebot/__init__.py", line 2523, in send_media_group
    result = apihelper.send_media_group(
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/telebot/apihelper.py", line 516, in send_media_group
    return _make_request(
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/telebot/apihelper.py", line 162, in _make_request
    json_result = _check_result(method_name, result)
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/telebot/apihelper.py", line 189, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 400. Description: Bad Request: too many messages to send as an album
